contracts/REXMarket.sol

```diff
@@ -740,6 +737,10 @@ abstract contract REXMarket is Ownable, SuperAppBase, Initializable {
             _agreementData, _superToken
         );
 
+        // Enforce speed limit on flowRate
+        require(uint128(uint(int(_flowRate))) % market.outputPools[market.outputPoolIndicies[_superToken]].shareScaler == 0, "notScaled");
+        require(uint128(uint(int(_flowRate))) > market.outputPools[market.outputPoolIndicies[_superToken]].shareScaler * 1e3, "tooLow");
```

Comment:
This enforces the speed limit, slower than this rate can lead to rounding errors when performing share scaling